pybikes/bysykkel.py

```diff
@@ -0,0 +1,86 @@
+# -*- coding: utf-8 -*-
+import json
+
+from .base import BikeShareSystem, BikeShareStation
+from . import utils
+
+
+class BySykkel(BikeShareSystem):
+
+    meta = {
+        'system': 'BySykkel',
+        'company': ['Urban Infrastructure Partner']
+    }
+
+    def __init__(self, tag, meta, feed_url, feed_details_url, feed_token):
+        super(BySykkel, self).__init__(tag, meta)
+        self.feed_url = feed_url
+        self.feed_details_url = feed_details_url
+        self.feed_token = feed_token
+
+    def update(self, scraper=None):
+        if scraper is None:
+            scraper = utils.PyBikesScraper()
+            scraper.setUserAgent("pybikes - github.com/eskerda/pybikes/tree/master/pybikes")
```

Comment:
User agent should not be specified (ie: should be the default) unless it's necessary for the spider to actually work. Unless, of course, it's specified on the terms for https://developer.oslobysykkel.no/api